packages/nova-react/src/localization/nova-localization.test.tsx

```diff
@@ -24,35 +20,27 @@ describe(useFormat, () => {
   it("is able to access the `format` function provided by the provider", () => {
     expect.assertions(3);
 
-    const formatFn = jest.fn();
+    const formatFn = vi.fn();
     const localization: NovaLocalization = {
-      useFormat: jest.fn().mockReturnValue(formatFn),
+      useFormat: vi.fn().mockReturnValue(formatFn),
     };
-
```

Comment:
No love for new lines? 🤷

Reply:
Turns out agent doesn't love them when doing automated refactor, can try to bring most of them back tough